src/components/benefitsHub/query.ts

```diff
@@ -68,5 +69,6 @@ export const formatter: QueryFormatter<NodeLandingPage, BenefitsHub> = (
     titleIcon: entity.field_title_icon,
     spokes: spokes,
     fieldLinks: fieldLinks,
+    fieldSupportServices: entity.field_support_services,
```

Comment:
The pattern here is to also format the fields in this object so we're not mixing camelCase with snake_case fields confusingly. The way we usually do that is by making and using another `formatter()` function for the entity type that this is.